runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -191,16 +193,22 @@ public ItemComposition getItemComposition(int itemId)
 	 * @param quantity
 	 * @return
 	 */
-	public static String quantityToStackSize(int quantity)
+	public static String quantityToStackSize(long quantity)
 	{
-		if (quantity >= 10_000_000)
+		// Billion to show up after 10,000M (similar to 10,000K), since max stacks show 2147M
+		if (quantity >= 10_000_000_000L)
+		{
+			return quantity / 1_000_000_000L + "B";
```

Comment:
Don't add bills as then the quantity will be really inacurrate and we are following OSRS naming with this method, and it doesnt have bills, only K and M i believe